sig-robotics/README.md

```diff
@@ -7,11 +7,34 @@ The [charter](charter.md) defines the scope and governance of the Robotics Speci
 
 ## Meetings
 
-
-* Regular SIG Meeting: [Tuesday at 10:00 UTC+8](https://zoom.us/my/kubeedge) (weekly, starts on 22th, June 2021). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
- * [Meeting notes and Agenda]().
- * [Meeting Calendar]() | [Subscribe]()
+* Regular SIG Meeting: [Wednesday at 10:00 UTC+8](https://zoom.us/my/kubeedge) (biweekly, starts on 22th, June 2021). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
+ * [Meeting notes and Agenda](https://docs.google.com/document/d/1i-DTZTsj2GjKHlocpkyzfEO--zYvx7sZ4bEFoJCy4YM/edit#).
+ * [Meeting Calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MGRnMGJzdjhmY3E2aDhhOTZlOXV0M211dXBfMjAyMzAyMDdUMDIwMDAwWiBodWFuZzY4OTRAbQ)
 
 ## Contact
-- Slack: [#sig-robotics]()
+
+- [#sig-Robotics slack channel](https://join.slack.com/t/kubeedge/shared_invite/enQtNjc0MTg2NTg2MTk0LWJmOTBmOGRkZWNhMTVkNGU1ZjkwNDY4MTY4YTAwNDAyMjRkMjdlMjIzYmMxODY1NGZjYzc4MWM5YmIxZjU1ZDI)
```

Comment:
Do we actually have this dedicated channel named `sig-Robotics`? i cannot find KubeEdge slack...